### DISCUSSION TITLE
how to debug CKAN in a Docker container using Visual Studio Code Debugger?

### DISCUSSION THREAD

**Discussion by kumarvivek1752** [ckan/ckan#7785] [Q&A]
hi everyone I'm fetching issue while debugging Ckan with vs code debugger. I installed ckan via docker

**Comment by glacier1213** [comment]
Hi, I am currently working on this. I am trying to figure out how to debug the code in the Docker container with VS Code Debugger. Have you figured it out?

**Reply by kumarvivek1752** [reply]
Yes, i used pdb.

**Comment by glacier1213** [comment]
Thank you. But I feel like using pdb is a bit inconvenient.
